packages/graphql-transformer-core/src/tableNameMap.ts

```diff
@@ -0,0 +1,24 @@
+import { InvalidDirectiveError } from '.';
```

Comment:
We should try not to pollute the core with transformer plugin specific things. Would it be possible to move all this logic to the plugin itself

Reply:
I agree this is a bit jank. It could be moved to the mapsTo transformer but that poses a few different issues:
1. transformer plugins don't handle parsing the schema, they are just given ast nodes and act on them
2. This function is called by the function category plugin and the awscloudformation provider plugin. It doesn't seem like those packages should depend directly on a transformer plugin

A more "proper" way to fix this issue would be to store the model name mapping as an output of the transformer somewhere and then read that in when needed in other places. That seemed like too much refactor work at this time, but maybe you feel differently?